seed/ts-sdk/api-wide-base-path/README.md

```diff
@@ -18,7 +18,7 @@ Instantiate and use the client with the following:
 ```typescript
 import { SeedApiWideBasePathClient } from "@fern/api-wide-base-path";
 
-const seedApiWideBasePath = new SeedApiWideBasePathClient({ environment: "YOUR_BASE_URL" });
+const seedApiWideBasePath = new SeedApiWideBasePathClient();
```

Comment:
My change conflicted with @amckinney's change here: https://github.com/fern-api/fern/pull/3850 which is the likely cause of this seeming regression. Investigating...